### PR TITLE
Update entries for '81.169.246.181'

### PR DESCRIPTION
**Pull request by LipkeGu**

### CODE REVIEW

File: content/packages/cnc-mirrors.txt [hunk 1]
```diff
@@ -2,5 +2,5 @@ http://openra.ppmsite.com/cnc-packages.zip
 http://srvdonate.ut.mephi.ru/openra/cnc-packages.zip
 http://openra.baxxster.no/openra/cnc-packages.zip
 http://openra.us.baxxster.no/openra/cnc-packages.zip
-http://81.169.246.181/cnc-packages.zip
+http://81.169.246.181/openra/cnc/cnc-quickinstall.zip
```
Comment (Mailaender) [inline content/packages/cnc-mirrors.txt]: You added this one to the wrong list, which will cause problems.

**Comment by LipkeGu** [comment]
updated

**Comment by LipkeGu** [comment]
updated

File: content/packages/ts-music-mirrors.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 http://openra.baxxster.no/openra/ts-music.zip
 http://openra.us.baxxster.no/openra/ts-music.zip
-http://81.169.246.181/ts-music.zip
+http://81.169.246.181/openra/ts/ts-tsmusic.zip
```
Comment (Mailaender) [inline content/packages/ts-music-mirrors.txt]: This now gives "404 - File or directory not found." Shall we remove it or can you fix it server side?

**Pull request merged**